pkg/sql/logictest/testdata/logic_test/interval

```diff
@@ -367,3 +367,14 @@ subtest regression_62369
 
 query error "10000000000000000000000000000000000": value out of range
 SELECT INTERVAL '10000000000000000000000000000000000 year'
+
+query T
+SELECT i / 2 FROM ( VALUES
+  ('0 days 0.253000 seconds'::interval),
+  (INTERVAL '0.000001'::interval),
+  (INTERVAL '0.000002'::interval)
```

Comment:
by all the way up to 0.00001, i meant 0.000003, 0.000004, 0.000005, etc as well!

Reply:
from 0.000001 to 0.000009?